iquip/apps/scheduler.py

```diff
@@ -0,0 +1,330 @@
+        self.model.expData.sort(key = lambda x: x.arginfo["priority"], reverse = True)
+
+    def changeExp(self, idx: int, info: ExperimentInfo):
+        """Changes the information of the particular experiment to given information.
+
+        Args:
+            idx: The index of to-be-changed experiment.
+            info: The experiment information.
+        """
+        if info:
+            self.model.expData[idx].changeInfo(info)
+        else:
+            self.delExp(self.model.expData[idx])
+
+    def delExp(self, info: ExperimentInfo):
+        """Deletes the experiment from 'queued experiments' section (expList).
+
+        Args:
+            info: The experiment information.
+        """
+        self.model.expData.remove(info)
+
+
+class ExperimentModel(QAbstractListModel):
+    """Model for managing the data in the experiment list.
+    
+    Attributes:
+        data: The list of ExperimentView widget.
+    """
+
+    def __init__(self, data: list, parent=None):
+        super().__init__(parent)
+        self.expData = data
+
+    def rowCount(self, parent=QModelIndex()):
+        """Overrided."""
+        return len(self.expData)
+
+    def data(self, index: QModelIndex, role=Qt.DisplayRole):
+        """Overrided."""
+        return self.expData[index.row()]
+
+    def supportedDropActions(self):
+        """Overrided."""
+        return Qt.CopyAction | Qt.MoveAction
+
+    def mimeData(self, index: QModelIndex) -> QMimeData:
+        """Fetches the index of the selected element.
+        
+        Args:
+            index: The ModelIndex instance containing 
+                   row, column, and etc. values of the selected element.
+        """
+        mime = super().mimeData(index)
+        mime.setText(str(index[0].row()))
+        return mime
+
+    def dropMimeData(self,
+        mimedata: QMimeData,
+        action: QAction,
+        row: int,
+        column: int,
+        parentIndex: QModelIndex):
+        """Changes the priority of the experiments.
+        
+        Args:
+            mimedata: The QMimeData instance containing str value of the index.
+            action: The QtAction instance classifying the action 
+                    (for terminating the function when it is not dropped in the appropriate region)
+            row: The target row that is to be changed with the experiment in mimedata.
+            column: The target column, which is set to zero as it is a QListView.
+            parentIndex: The ModelIndex instance containing
+                         row, column, and etc. values of the target element.
+        """
+        idx = int(mimedata.text())
+        if action == Qt.IgnoreAction:
+            return True
+        taridx = row if row < self.rowCount() else -1
+        if self.expData[idx].arginfo["priority"] != self.expData[taridx].arginfo["priority"]:
+            return True
+        if idx > row:
+            self.expData = self.expData[:row] + [self.expData[idx]] + \
```

Comment:
Please don't use a backslash referred to [PEP 3125](https://peps.python.org/pep-3125/).
Instead, you can use parentheses.